src/main/java/com/easyrest/pages/moderatorPanel/Owners.java

```diff
@@ -0,0 +1,44 @@
+package com.easyrest.pages.moderatorPanel;
+
+import com.easyrest.components.admin.ActionOnUser;
+import com.easyrest.components.moderator.LeftSideMenu;
+import com.easyrest.pages.BasePage;
+import org.openqa.selenium.WebDriver;
+
+public class Owners extends BasePage {
+
+   private LeftSideMenu leftSideMenu = new LeftSideMenu(driver);
+   private ActionOnUser actionOnOwner = new ActionOnUser(driver);
+
+    public Owners(WebDriver driver) {
+        super(driver);
+    }
+
+    public Owners clickActiveOwners() {
+        leftSideMenu.clickOnMenuButtonOwners();
```

Comment:
`leftSideMenu.clickOnMenuButtonOwners();` returns an instance of Owners page but you don't use it. Why don't you use it?